pkg/knuu/knuu_old.go

```diff
@@ -96,7 +96,7 @@ func InitializeWithScope(testScope string) error {
 	}
 
 	// TODO: this must be moved to somewhere more meaningful
-	tmpKnuu.HandleStopSignal()
+	// tmpKnuu.HandleStopSignal()
```

Comment:
why this line is commented out? does it create any problems for this feature?

Reply:
For my workflow, it's annoying to wait for the deletion to complete. I will enable it again.

Instead of cleaning up like that, could we deploy a timeout handler without a time specified so that the cleanup is async?

Reply:
I think to have a better structured tests, we can use test suit and add the cleanup in the tear down func. Also this handle signal can be added by the user as well. So currently the new knuu obj does not call it by default.